pkg/git/provider_ado.go

```diff
@@ -97,7 +100,10 @@ func (g *adoGit) GetDefaultBranch(ctx context.Context, orgRepo string) (string,
 	if err != nil {
 		return "", err
 	}
-
+	if r.DefaultBranch == nil {
+		// Repo is empty.
+		return "main", nil
```

Comment:
is there a way to get the "default default branch" from the organization level? is there such a setting in azure-devops?
preferably, we would not default to hard-coded "main". but i guess if there is no org-level setting, it might solve the problem for the majority of uses.

another question - if the repo is empty, and the code returns "main", does the rest of the code works? does it successfully commit to main and push it to origin?